docs/src/modules/brandingTheme.ts

```diff
@@ -464,17 +475,21 @@ export function getThemedComponents(): ThemeOptions {
         },
         styleOverrides: {
           root: ({ theme, ownerState }) => ({
+            transition: 'all 120ms ease-in',
+            '&:active': { transform: 'scale(.96)' },
```

Comment:
Are we sure about this? This feels better on my end:

```suggestion
```

Before:

https://github.com/mui/material-ui/assets/3165635/9466a227-fa66-4404-b69c-88bc1c23d595

After:

https://github.com/mui/material-ui/assets/3165635/0c0e9db1-7273-47e7-99fe-b6c33f049f62

Reply:
Could be — how much did you put in your "after" example? It doesn't seem like you removed it entirely.

Reply:
The after uses the code diff, it rely on ButtonBase style.

Reply:
I've removed the scale effect from both the Button Base and buttons on a later PR to avoid any issues with text and elements rasterizing.